docs/Makefile

```diff
@@ -16,12 +16,12 @@ help:
 
 .PHONY: help Makefile generate-docs
 
-$(ANSIBLEGIT):
-	git clone --depth=1 https://github.com/ansible/ansible/ -b stable-2.9 $(ANSIBLEGIT)
-
-generate-docs: $(ANSIBLEGIT)
-	rm -rf ./modules/
-	bash -c "source $(ANSIBLEGIT)/hacking/env-setup && ANSIBLE_COLLECTIONS_PATHS=../build/collections python3 $(ANSIBLEGIT)/hacking/build-ansible.py document-plugins --module-dir ../plugins/modules --template-dir ./_templates --template-dir $(ANSIBLEGIT)/docs/templates --type rst --output-dir ./modules/"
+generate-docs:
+	mkdir -p ./plugins/
+	rm -rf ./plugins/*
+	ANSIBLE_COLLECTIONS_PATHS=../build/collections antsibull-docs collection --use-current --dest-dir ./plugins/ theforeman.foreman
+	mv ./plugins/collections/theforeman/foreman/* ./plugins/
+	rm -rf ./plugins/collections/
```

Comment:
Is there a way we can do without all this copying, deleting, moving around?

Reply:
Only if we accept to have those (for us) useless subdirs.

antsibull currently focuses on building docs for *multiple* collections, so that structure is useful there.

Reply:
If we can make them appear in `build`, i don't care. They would also be subject to `make clean`.

Reply:
Yeah, we can dump that into build